src/sage/graphs/connectivity.pyx

```diff
@@ -732,13 +734,173 @@ def blocks_and_cuts_tree(G):
     return g
 
 
+def is_edge_cut(G, edges):
+    """
+    Check whether ``edges`` is an edge cut of ``G``.
+
+    A set of edges is an edge cut of a graph if its removal increases the number
+    of connected components. In a digraph, we consider the number of (weakly)
+    connected components.
+
+    This method is not working for (di)graphs with multiple edges. Furthermore,
+    edge labels are ignored.
+
+    INPUT:
+
+    - ``G`` -- a Sage (Di)Graph
+
+    - ``edges`` -- an iterable container of edges
+
```

Comment:
whenever possible, just explain the inputs in plain mathematical terms.

"a (di)graph" instead of "a Sage (Di)Graph".

"a list of edges" instead of "an iterable container of edges".

Reply:
Then you may say "a set of edges". A method that accepts a python set would also accept a python list, an iterator, and the like.